plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/runtime/DBRCancellable.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.model.runtime;
+
+public interface DBRCancellable {
```

Comment:
Do we need this interface?

Reply:
We use it to have cancel method for RunnableWithResult

Reply:
I mean, do we really need the interface? Can be keep just the method itself?

Reply:
yeah, you're right, we can add this method to RunnableWithResult directly